From 81a929ac141aae66b6450e8ce93cb357ed404cda Mon Sep 17 00:00:00 2001
From: Timo Gurr <timo.gurr@gmail.com>
Date: Mon, 1 Jun 2015 16:10:16 +0200
Subject: [PATCH] configure: Fix configure failure when building without
 gstreamer support

Error introduced in 20ea22e0a11a9bdfe4d8125b68083249b694338a, resulting in a
configure/build error when building without gstreamer:

configure: error: conditional "HAVE_GST_CHECK" was never defined.
Usually this means the macro was only invoked conditionally.

https://bugs.freedesktop.org/show_bug.cgi?id=90801
Signed-off-by: Gustavo Zacarias <gustavo@zacarias.com.ar>
---
Patch status: upstream

 configure.ac | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/configure.ac b/configure.ac
index 6031cec..64a571f 100644
--- a/configure.ac
+++ b/configure.ac
@@ -231,9 +231,6 @@ AS_IF([test "$with_gstreamer" != no], [
 		[
 		have_gst_check=no
 		])
-
-	AM_CONDITIONAL(HAVE_GST_CHECK, test "$have_gst_check" = yes)
-
 ])
 
 AS_IF([test "$with_gstreamer010" != no], [
@@ -260,6 +257,7 @@ AC_SUBST(gstplugindir)
 AC_SUBST(gstplugin010dir)
 
 AM_CONDITIONAL(WITH_GSTREAMER, test "$with_gstreamer" = yes)
+AM_CONDITIONAL(HAVE_GST_CHECK, test "$have_gst_check" = yes)
 AM_CONDITIONAL(WITH_GSTREAMER010, test "$with_gstreamer010" = yes)
 
 GUPNP_IGD_REQUIRED=0.2.4
-- 
2.10.2

